javalib/src/main/scala/java/util/regex/Matcher.scala

```diff
@@ -97,7 +97,8 @@ final class Matcher private[regex] (
           while (i < len && isDigit(replacement.charAt(i)))
             i += 1
           val group = Integer.parseInt(replacement.substring(j, i))
-          sb.append(this.group(group))
+          val replaced = this.group(group)
+          if (replaced != null) sb.append(replaced)
```

Comment:
According to our coding style, side-effecting branches of an if must always be on their own line:
```suggestion
          if (replaced != null)
            sb.append(replaced)
```